src/Adapter/Http/ApachePassword.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Authentication\Adapter\Http;
+
+use function base64_encode;
+use function chr;
+use function count;
+use function explode;
+use function hash_equals;
+use function mb_strlen;
```

Comment:
Should we add `ext-mbstring` to composer? It seems like it's the first place we're adding `mb_*` functions.